src/main/java/application/MenuController.java

```diff
@@ -129,6 +128,7 @@ public EntityDetailListResponse getDetails(@RequestBody SessionNavigationBean se
                 sessionNavigationBean.getSearchText()
         );
         Screen currentScreen = menuSession.getNextScreen();
+        menuSessionRepo.save(new SerializableMenuSession(menuSession));
```

Comment:
this'll save many more menu sessions. i think a follow up item for this is to figure out a way to delete them periodically